.github/workflows/beam_Inference_Python_Benchmarks_Dataflow.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           arguments: |
             -PloadTest.mainClass=apache_beam.testing.benchmarks.inference.pytorch_image_classification_benchmarks \
             -Prunner=DataflowRunner \
-            -PpythonVersion=3.9 \
+            -PpythonVersion=3.10 \
```

Comment:
Can we manually run these on this branch - https://github.com/apache/beam/actions/workflows/beam_Inference_Python_Benchmarks_Dataflow.yml

Reply:
I had to port the workflow changes to a mainline Beam repo branch, but that run is going now - https://github.com/apache/beam/actions/runs/11019905239

Reply:
Clean run, merging